desktop/components/commercial_filter/views/pillbox/index.styl

```diff
@@ -1,3 +1,9 @@
+html[data-lab-features*='Keyword Search']
```

Comment:
Because we put all lab features in the DOM (as a top-level data attribute on the page), it makes conditionally styling elements easy using this type of selector.

When we replace with the actual AB test, I can similarly add AB tests into the DOM to support this type of selector/styling. I'm actually surprised that's not the case already, that seems very useful for AB tests in particular (like AB testing the color of a single button).

Reply:
I wasn't aware of this pattern... makes sense to me.

Reply:
Good idea to use this for A/B tests as well, absolutely.